src/contour/Config.cpp

```diff
@@ -1698,6 +1698,16 @@ std::optional<actions::Action> YAMLConfigReader::parseAction(YAML::Node const& n
                 return std::nullopt;
         }
 
+        if (holds_alternative<actions::CreateSelection>(action))
+        {
+            if (auto chars = node["chars"]; chars.IsScalar())
```

Comment:
```suggestion
            if (auto chars = node["delimiters"]; chars.IsScalar())
```

I think here (and all the other related places), the JSON config key for custom delimiters should be named `delimiters` rather than `chars`, to make it more intuitive what the contents of that configuration key actually is.